test/utility/parse_opts.c

```diff
@@ -47,10 +47,73 @@ extern int __wt_optreset;
     } while (0)
 
 /*
- * parse_tiered_opt --
+ * parse_init_random --
+ *     Initialize the random number generator from the seed. If the seed is not yet set, get a
+ *     random seed.
+ */
+static void
+parse_init_random(WT_RAND_STATE *rnd, uint64_t *seedp)
+{
+    if (*seedp == 0) {
+        __wt_random_init_seed(NULL, rnd);
+        *seedp = (rnd->v & 0xffff);
+    }
+    rnd->v = *seedp;
+}
+
+/*
+ * parse_seed --
+ *     Parse a random number generator seed from the command line.
+ */
+static void
+parse_seed(uint64_t *seedp, const char *seed_str, char **parse_end)
+{
+    *seedp = (uint64_t)strtoll(seed_str, parse_end, 10);
+}
+
+/*
+ * parse_tiered_random_seeds --
  *     Parse a command line option for the tiered storage configurations.
  */
 static void
+parse_tiered_random_seeds(TEST_OPTS *opts, const char *seed_str)
+{
+    char *parse_end;
+    const char *seed_end;
+    static const char *SEED_USAGE =
+      "-PS parameter is comma separated write and read seeds, e.g. -PSW1234,R5678";
```

Comment:
Can these be talked about and presented in alphabetical order? "...separated read and write seed, e.g. -PSR1234,W5678"?